test/Autofac.Integration.AspNetCore.Multitenant.Test/Autofac.Integration.AspNetCore.Multitenant.Test.csproj

```
@@ -2,7 +2,7 @@
 
   <PropertyGroup>
     <TargetFrameworks>netcoreapp2.1;netcoreapp3.1;net5.0</TargetFrameworks>
-    <NoWarn>$(NoWarn);CS1591</NoWarn>
+    <NoWarn>$(NoWarn);CS1591; SA1600; SA1633</NoWarn>
```

```suggestion
    <NoWarn>$(NoWarn);CS1591;SA1600</NoWarn>
```
I think we want headers on everything, including tests, don't we?

---

Addresses with latest commit.